roxie/ccd/ccdmain.cpp

```diff
@@ -1129,10 +1132,44 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
                 if (port)
                 {
                     const char *protocol = roxieFarm.queryProp("@protocol");
+                    const char *passPhrase = nullptr;
+                    const char *certFile = nullptr;
+                    const char *keyFile = nullptr;
```

Comment:
Not important, but I still think these can be moved to directly say:

`const char *certFile = roxieFarm.queryProp("@certificateFileName");`
and
`const char *keyFile =  roxieFarm.queryProp("@privateKeyFileName");`

where used below.